packages/contracts-bridge/contracts/test/TokenRegistry.t.sol

```diff
@@ -490,10 +478,7 @@ contract TokenRegistryTest is BridgeTestFixture {
         vm.assume(domain != homeDomain && domain != 0);
         address repr = tokenRegistry.getRepresentationAddress(domain, id);
         assertEq(repr, address(0));
-        address calculated = computeCreateAddress(address(tokenRegistry), 2);
-        // test event emmission
-        vm.expectEmit(true, true, true, false);
-        emit TokenDeployed(domain, id, calculated);
+
```

Comment:
was this removed on purpose?

Reply:
The tests should not be green in this one. Checking.